### DISCUSSION TITLE
Recommended/validated Postgres version (as version 12 will be end of support in approx. 6 months)

### DISCUSSION THREAD

**Discussion by GOllagnon** [frePPLe/frepple#657] [Q&A] [answered]
Dear Forum / Johan,

I am preparing the server of frePPLe in the context of next release 9.0.0 to come, and I have a question related to versions of Postgres database, as version 12 will have the final release in approx. 6 months.

It is indicated in the Installation guide / Linux binaries section: **_"Install postgreSQL 12 or higher"_**, so is it possible to use version 13 vs. 14 vs. 15 vs. 16?
Are some of them more recommended than others?
Or maybe some have already been qualified vs. others not, so I can for example choose 16 (e.g. if only 13, 14, and 15 have been qualified) and let you know is some strange behaviors occur?

What is your recommandation (if any) or is it possible to use any version?

Thanks & BR,

Guy.

**Comment by jdetaeye** [comment, marked as answer]
Any version higher than the stated minimum will work. The recommended version is the latest official version, which is 16 at the moment.
